codex/uberdev-codex/lib/status.sh

```
@@ -0,0 +1,1028 @@
+    _uberdev_status_note "read from: $root  (pointer $_UBERDEV_STATUS_GOAL_POINTER_BASENAME -> $goal_id)"
+    _uberdev_status_render_goal "$role" "$root" "$goal_id"
+  done <<EOF
+$_UBERDEV_STATUS_ROOTS
+EOF
+  if [ "$found" -eq 0 ]; then
+    _uberdev_status_note "no $_UBERDEV_STATUS_GOAL_POINTER_BASENAME under any readable root — no /goal run to resume."
+  fi
+}
+
+_uberdev_status_render_goal() {
+  local role="$1" root="$2" goal_id="$3" sidecar="$root/goal-$goal_id-runstate"
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)